KeyboardSpacer.js

```diff
@@ -6,11 +6,11 @@ import {
   Keyboard,
   LayoutAnimation,
   View,
-  ViewPropTypes,
   Dimensions,
   Platform,
   StyleSheet
 } from 'react-native';
+import {ViewPropTypes} from 'deprecated-react-native-prop-types'
```

Comment:
Any chance we can just get rid of it all together? 😅

Reply:
This one is small, we should be able to do so.